### ISSUE TITLE
Looking for the best sweet shop in Chennai?

### ISSUE THREAD

**Issue by shubhfoods** [shubhfoods/Looking-for-the-best-sweet-shop-in-Chennai-#1]
If you are a sweet lover like me and want to satisfy your taste buds with the best sweets all around India, then your search ends here. In February 2020, when I was in Chennai, I found this amazing shop known as [Shubh Mitthai](https://shubhfoods.co.in/). This shop offers a large variety of delicacies that are made of natural ingredients and are totally safe because no artificial food coloring is used and all the sweets are made with pure ghee, which gives them a delectable taste, rich aroma, and freshness.

**Need sweets for special occasions?**

If yes, then you can get the best sweets and savories from Shubh Mithai and make your special occasions memorable with the divine taste of varieties of sweets. They even offer discounts on the bookings for your special events or corporate orders, which is like the cherry on the cake.

The divine taste of sweets like Kaju katli, Rajbhog, Kaju royal, Khoya Kaju roll, Kaju ghujia, Chocolate burfi, Ghewar, Rose chikki, and etc is surely going to satisfy your taste buds.

So, if there is any occasion like an intimate family gathering or there is any corporate event, you can make it special with the authentic taste of regional cuisines too.

**Homemade Chocolates:**

With sweets and savories, Shubh Mithai has come up with the brilliant idea of delivering homemade chocolates made with 100% cocoa and pure cow’s milk. These rich, creamy and pure homemade chocolates are the best option for gifting too. You can make someone else’s day special by giving them Shubh’s homemade chocolates which will leave anyone wanting more.

**Snacks**:

Snacks like Momos, Samosa, Bread Pakoda, Khasta Kachori, Pani Poori are so mouth-watering and yummy that will compel you to visit this shop again and again. And the Chaats here, just fabulous whether it is Basket Chaat or Dahi Gujhiya chaat, Spicy Aloo Chaat or Papdi Chaat, it will definitely leave you speechless.

In short, if want to experience anything from sweets to snacks, you can visit this shop in Nungambakkam and I can assure you that Shubh Mithai is the best sweet shop in Chennai. Below are the details about this shop, you can check it out, if interested.

Shop Name: Shubh Mithai
Website: https://shubhfoods.co.in/
Address: New no.12/1&2, Old no.53/1&2, Sterling Ave, Nungambakkam, Chennai, Tamil Nadu 600034

Mobile Number: 7603902445
